src/workers/search/get-matches.js

```diff
@@ -20,8 +20,17 @@ export default function getMatches(
   for (let i = 0; i < lines.length; i++) {
     let singleMatch;
     const line = lines[i];
+    let previousLastIndex = regexQuery.lastIndex;
     while ((singleMatch = regexQuery.exec(line)) !== null) {
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec#Finding_successive_matches) this should work fine but I found that in some cases this is creating an infinite loop because `regexQuery.exec()` is not updating `regexQuery.lastMatch` and it keeps returning the same match over and over again.  Is this a bug in the javascript engine?